### PR TITLE
Feat badge

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Added Badge Component
- Badge on Avatars
- Badge on Icons

image
image

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    font-size: 1.8rem;
+    border: 3px solid var(--background-color);
+    padding: 1px;
+}
+
+.count-badge {
+    font-size: 1rem;
+    border-radius: 50%;
+    position: absolute;
+    top: -20px;
```
Comment (Jia-A) [inline components/badge.css]: This can be -1.25rem if you want it responsive I guess.

**Review by Jia-A — COMMENTED** [review]
Everything looks good and ready to merge.

**Review by ZuberDunge — COMMENTED** [review]
Excellent code structure, Badge design looks good too. Well done, Himanshi. 

Good to go from my side.

File: components/badge.css (hunk 1 above)
Comment (hinanshisuthar) [reply components/badge.css]: Yes cool. Would add that.

**Review by Jia-A — COMMENTED** [review]
Amazing!!!

**Pull request merged**